src/Compilers/Core/CodeAnalysisTest/DefaultAnalyzerAssemblyLoaderTests.cs

```diff
@@ -1197,24 +1239,25 @@ public void AssemblyLoading_NativeDependency(bool shadowLoad)
                 var analyzer = analyzerAssembly.CreateInstance("Class1")!;
                 var result = analyzer.GetType().GetMethod("GetFileAttributes")!.Invoke(analyzer, new[] { testFixture.AnalyzerWithNativeDependency.Path });
                 Assert.NotEqual(INVALID_FILE_ATTRIBUTES, result);
-                Assert.Equal(FileAttributes.Archive, (FileAttributes)result!);
+                Assert.Equal(FileAttributes.Archive | FileAttributes.ReadOnly, (FileAttributes)result!);
```

Comment:
>| FileAttributes.ReadOnly [](http://example.com/codeflow?start=52&length=25)

What was the change that affected `GetFileAttributes()`?

Reply:
Looks like it might be `GenerateDll()`.

Reply:
Yes that is the change that caused this.